TSC-Charter.md

```diff
@@ -84,25 +63,25 @@ including:
 * Conduct guidelines.
 * Maintaining the list of additional Collaborators.
 * Development process and coding standards.
-* Mediating technical conflicts between Collaborators or ASC Organization
+* Mediating technical conflicts between Collaborators or Planetary Software Organization
 projects.
 
-The TSC will define ASC Software Organization's release vehicles and serve as
-ASC Software’s primary technical liaison body with external open
+The TSC will define the Planetary Software Organization's release vehicles and serve as
+Planetary Software’s primary technical liaison body with external open
 source projects, consortiums and groups.
 
-## Section 6. ASC Software Organization Operations.
+## Section 6. Planetary Software Organization Operations.
 
-The TSC will establish and maintain a development process for ASC Software Organization's Projects. The development process will establish guidelines
+The TSC will establish and maintain a development process for the Planetary Software Organization's Projects. The development process will establish guidelines
 for how the developers and community will operate. It will, for example,
 establish appropriate timelines for TSC review (e.g. agenda items must be
 published at least a certain number of hours in advance of a TSC
 meeting).
 
-There will be multiple Projects under the ASC Software Organization organized by
+There will be multiple Projects under the Planetary Software Organization organized by
 modules or subsystems. The TSC is responsible for organizing the Project
 structure, including possibly the creation and alignment of sub-Projects.
```

Comment:
Should the language here be more along the lines of 'including possibly supporting the creation and alignment of sub-Projects'. The heritage of this doc coming from Node with such a homogeneous environment comes through here pretty strongly.

Reply:
Yes, I was just going through these during while half-paying attention to another meeting and trying to figure out what the minimal changes were to 'flip' the document.  I wasn't parsing deeply.  I think a change like that would be good.